### DISCUSSION TITLE
flagged issue

### DISCUSSION THREAD

**Discussion by mdAshik922** [community/community#38069] [New to GitHub]
How can I solve the **flagged** issue? please

**Comment by david-engelmann** [comment]
What is the flagged issue?

**Reply by mdAshik922** [reply]
It hides the account so that it is not found when searching.

**Reply by david-engelmann** [reply]
This has been reported a few times now. If you hit enter from the search bar, the query should process. The amount of times it's been submitted makes me think it will be resolved soon

**Reply by martins-avots** [reply]
I currently have this issue, too,
but I also did not receive any feedback in GitHub support tickets.
